rever/activities/travis_ci.xsh

```diff
@@ -0,0 +1,99 @@
+"""Activity for keeping a changelog from news entries."""
+import os
+import re
+import sys
+
+from xonsh.tools import print_color
+
+from rever import vcsutils
+from rever.activity import Activity
+from rever.tools import eval_version, replace_in_file
+
+TRAVIS_CI = """sudo: False
+
+language: python
+
+matrix:
+  include:
+    - python: 3.6
+
+install:
+  # Install conda
+  - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
+  - bash miniconda.sh -b -p $HOME/miniconda
+  - export PATH="$HOME/miniconda/bin:$PATH"
+  - conda config --set always_yes yes --set changeps1 no
+  - conda config --add channels conda-forge
+  - conda update conda
+
+  # Install dependencies
+  - conda install {conda_deps} -c (conda_channels)
```

Comment:
Maybe also add a `{conda_extra}` so that users can add extra stuff to their conda install.

Reply:
What would be the environmental variable that determines that? Since the $DOCKER_CONDA_DEPS is an array wouldn't the user already add whatever they wanted to install into here?

Reply:
Yeah, so I am suggesting something a little more sophisticated. Basically we could add new global env vars `CONDA_DEPS`, `APT_DEPS`, `PIP_DEPS`, etc. and then if `DOCKER_*_DEPS` or the `TRAVIS_*_DEPS` are not found, they fall back to the value set in the global env vars.

Reply:
You could make an env var if you wanted. It might be better to separate those since you might be interested in the deps for something else.